### DISCUSSION TITLE
Why initial top is 1, not 0?

### DISCUSSION THREAD

**Discussion by dayo05** [gudzpoz/luajava#187] [Q&A] [answered]
```kt
val lua = LuaJit()

assertEquals(1, lua.top) // why??
    
for((a, b) in lua.get().entries)
    println(a.toJavaObject().toString() + " " + b.toJavaObject())
```

I made simple example of instance creation, `lua.top` returns 1 even LuaJit instance just constructed.

What is it?

the for loop prints:
```
__gc RefLuaValue$FUNCTION@party.iroiro.luajava.luajit.LuaJit@1169afe1:3
__eq RefLuaValue$FUNCTION@party.iroiro.luajava.luajit.LuaJit@1169afe1:5
__len RefLuaValue$FUNCTION@party.iroiro.luajava.luajit.LuaJit@1169afe1:7
__index RefLuaValue$FUNCTION@party.iroiro.luajava.luajit.LuaJit@1169afe1:9
__newindex RefLuaValue$FUNCTION@party.iroiro.luajava.luajit.LuaJit@1169afe1:11
```

**Comment by dayo05** [comment]
And also, can I just pop it?

**Reply by gudzpoz** [reply, marked as answer]
The value(s) on stack are unintended remains of Lua library initialization and should be safe to remove.
However, leaving the stack empty might trigger a bug I just found: `LuaValue::size` seemed to pop an extra item off the stack, which has been fixed by a new v4.0.1 version.
